tipg/model.py

```diff
@@ -86,7 +86,7 @@ class Collection(BaseModel):
     links: List[Link]
     extent: Optional[Extent]
     itemType: str = "feature"
-    crs: List[str] = ["http://www.opengis.net/def/crs/EPSG/0/4326"]
+    crs: List[str] = ["http://www.opengis.net/def/crs/OGC/1.3/CRS84"]
```

Comment:
The CRS property of the collection is `the list of coordinate reference systems supported by the service` so in TiPG it makes senses to have this set to `http://www.opengis.net/def/crs/OGC/1.3/CRS84` 

>   crs:
    description: the list of coordinate reference systems supported by the service
    type: array
    items:
      type: string
    default:
      - http://www.opengis.net/def/crs/OGC/1.3/CRS84
    example:
      - http://www.opengis.net/def/crs/OGC/1.3/CRS84
      - http://www.opengis.net/def/crs/EPSG/0/4326

ref: http://schemas.opengis.net/ogcapi/features/part1/1.0/openapi/schemas/collection.yaml